modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Veteran
+  module Service
+    class Representative < ActiveRecord::Base
+      self.primary_key = :representative_id
+
+      validates_presence_of :poa
```

Comment:
seems like since you have poa, which is essentially the pk on Organization, you could do a has_one type deal to get some rails goodness.

Reply:
Again this PR is intended to be only for creating the module and handling the migration but I can include the rails magic in my next PR where I'll add most of the business logic I've written.